docs/core-concepts/bot-sequences/actions.mdx

```diff
@@ -1,9 +1,624 @@
+- CVObjectDetection via ImageQuery selects the most common object in the query image. If the query image contains multiple objects, (such as a cat and a dog) it will only select one--whichever is more prominent.
+
+**Example: Image query example**
+
+```json
+{
+    "name": "CV Object Detection Action: Click on a tree using an image query",
+    "description":"Moves the mouse over the tree, then clicks and releases on the tree. Criteria waits for the action to complete.",
+    "endCriteria": [
+         {"type":"ActionComplete", "transient":false, "data":{}}
+    ],
+    "botAction":{    
+        "type":"Mouse_ObjectDetection",
+        "data": {
+            "imageQuery":"/9j/4AAQ...<rest of the base64 encoded image data>...VLj3ieS/9k=",
+            or
+            "imageQuery":"file://???/sample_images/tree.jpg",
+            or  
+            "imageQuery":"resource://sample_images/tree",
+            "actions": [
+                {"leftButton":false, "duration":0.1}, // Move the mouse to the center of the tree
+                {"leftButton":true, "duration":0.1},  // Click the left mouse button
+                {"leftButton":false, "duration":0.1}  // Release the left mouse button
+            ]
+        }
+    }
+}
+```
+- Using file:// path (??? represents the path to this folder on your system)
+- Using resoure:// path (note that the `sample_images` folder must be under a Resources folder in your project)
+
+**Example: Text query example**
+
+```json
+{
+    "name": "CV Object Detection Action: Click on a tree using a text query",
+    "description":"Moves the mouse over the tree, then clicks and releases on the tree. Criteria waits for the action to complete.",
+    "endCriteria": [
+         {"type":"ActionComplete", "transient":false, "data":{}}
+    ],
+    "botAction":{    
+        "type":"Mouse_ObjectDetection",
+        "data": {
+            "textQuery": "Tree",
+            "threshold": 0.4,
+            "actions": [
+                {"leftButton":false, "duration":0.1},  // Move the mouse to the center of the tree
+                {"leftButton":true, "duration":0.1},   // Click the left mouse button
+                {"leftButton":false, "duration":0.1}   // Release the left mouse button
+            ]
+        }
+    }
+}
+```
+
+### RestartGame
+
+This action type restarts the game. This should only be used as the 'last' segment in your sequence or segment list, and is useful for getting your game
+ready for another test.
+
+There is an interface you can implement to override the default behaviour of restarting the game. If this interface is not implemented when 
+the `RestartGame` Bot Action is used in a build of your game **outside the UnityEditor**, then the default action will be to call 
+`SceneManager.LoadScene(sceneBuildIndex: 0, mode: LoadSceneMode.Single)`.  **In the UnityEditor**, the action will **always** be to stop and re-enter play mode;
```

Comment:
We also destroy all 'dontdestroyonload' game objects before loading scene 0

Reply:
Added